grails-app/jobs/org/pih/warehouse/jobs/AssignIdentifierJob.groovy

```diff
@@ -20,13 +19,10 @@ class AssignIdentifierJob {
             return
         }
 
-        Product.withNewSession {
-            identifierService.assignProductIdentifiers()
-            identifierService.assignShipmentIdentifiers()
-            identifierService.assignReceiptIdentifiers()
-            identifierService.assignOrderIdentifiers()
-            identifierService.assignRequisitionIdentifiers()
-            identifierService.assignTransactionIdentifiers()
-        }
+//        Product.withNewSession {
```

Comment:
Note to self: test if we still need this!

Reply:
No unfortunately... I think I'm going to leave it as is for now, but I do think it's worth investigating in the future if this is actually a good idea. I'm thinking it'd be better to split them all into their own sessions/transactions.